frontend/catalyst/jax_tracer.py

```diff
@@ -273,6 +273,8 @@ class HybridOp(Operation):
             JAX primitive binder function to call when the quantum tracing is complete.
     """
 
+    visited = False
+
```

Comment:
```suggestion
```

Reply:
This one is important for the decomposition

Reply:
could you add a brief code comment here for the future reference?

Reply:
Yes for sure!

Reply:
Done

Reply:
Thanks!

Reply:
For each op, it checks if it is accepted (in the list of operations or an hybrid op that is visited), if it is not the case it decomposes recursively until it either have all accepted operations or fails if it is reaching the max expansion. The hybrid operation is handled slightly differently, the decomposition is to call `decompose` on tapes for each region, if it can decompose all of them then it succeed and we mark the operation as visited, which tells that we successfully expanded the hybrid op. Is it more clear?